cluster_tools/cluster_tools/_utils/meta.py

```diff
@@ -0,0 +1,34 @@
+import os
```

Comment:
why `meta.py`? I think, I'd find `misc.py` less confusing.

Reply:
I meant `meta` metaprogramming. Might also call it `metaprogramming` therefore, or do you prefer misc?

Reply:
Ah, I see, yes, metaprogramming would be clearer. However, I think, I'd prefer `reflection` in this case, as it fits even, (I think?) :)